packages/web-react/src/components/Accordion/AccordionHeader.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Icon } from '../Icon';
+import { AccordionHeaderProps } from '../../types';
+import { useAccordionStyleProps } from './useAccordionStyleProps';
+import { useAccordionContext } from './AccordionContext';
+import { useAccordionItemContext } from './AccordionItemContext';
+import { useAccordionAriaProps } from './useAccordionAriaProps';
+
+const AccordionHeader: React.FC<AccordionHeaderProps> = ({ children, slot }) => {
+  const { classProps } = useAccordionStyleProps();
+  const { toggle, isItemOpen, open } = useAccordionContext();
+  const { id } = useAccordionItemContext();
+
+  const itemToggle = () => {
+    if (toggle && id) {
+      toggle(id);
+    }
+  };
+
+  const isOpen = React.useMemo(
+    () => (id && isItemOpen && open ? isItemOpen(open, id) : false),
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [open, id],
+  );
+
+  const { triggerProps, headerProps } = useAccordionAriaProps({ id, isOpen });
+
+  return (
+    <h3 className={classProps.header} {...headerProps}>
```

Comment:
Missing `styleProps`. I think we should add them.

Reply:
This is only a sub component. We once solved something similar, but in the end we left styleProps only for the main and wrapping components.

Reply:
Same as here: https://github.com/lmc-eu/spirit-design-system/pull/594/files#r1043555993

Reply:
done